### DISCUSSION TITLE
[Feature] Disable video auto playback at web homepage

### DISCUSSION THREAD

**Discussion by MDsub** [immich-app/immich#6283] [Feature Request]
### The feature

My immich running on Unraid, media storage on HDD array, thumbnail on SSD cache pool. HDD array will auto spin down when it idle. If I open immich homepage and the move mouse on thumbnail, it will let HDD spin up. So I want to disable it.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by jenda69** [comment]
Similar configuration, same problem. I'd love to have an option to disable playback on hover.
Also I don't know whether Immich transcodes video also to low resolution for thumbnails by default, but I have transcoding disabled and this just starts downloading whole large video on hover which is not particularly great on slow or measured connections.

**Comment by samholton** [comment]
Created PR https://github.com/immich-app/immich/pull/7831. I added a user preference to disable this. But also added hover on play icon to play the thumbnail even when this preference is disabled - hopefully best of both worlds with keeping the playback target small enough to avoid accidental hovers.
